src/preload-hover.js

```diff
@@ -32,34 +32,34 @@ export default class PreloadHover {
       let timer;
       let uniqueLinks = uniqBy(links, 'href');
 
-      uniqueLinks.forEach(link => {
-        link.addEventListener('mouseover', () => {
+     domScope.addEventListener('mouseover', (e) => {
```

Comment:
`domScope` items could intersect resulting in duplicate `mouseover`/`mouseout` wireups for the same items. We can ignore this for now in favor of seeing what gains the library potentially affords our users.